CONTRIBUTING.md

```diff
@@ -14,7 +14,14 @@ See [GitHub's official documentation](https://help.github.com/articles/using-pul
 # Getting Started
 
 - Use LabVIEW 2015 or later.
-- Sync the source into the [LabVIEW 20xx]\ProjectTemplates folder.
+- Sync the source into the [LabVIEW 20xx]\ProjectTemplates folder in the following manner:
+	- NI_VeriStand_Consolidated.xml into the "ProjectTemplates\MetaData" folder
+	- The following items into the "ProjectTemplates\Source\NI VeriStand\Consolidated Custom Device" folder:
+		- NI_VS_Consolidated_CDW_MetaDataObj
+		- Template Classes
+		- Template Files
+		- Null.lvproj
+		-Consolidated Custom Device.lvproj (only necessary for development)
```

Comment:
The lack of a space between the - and "Consolidated" is causing it to render incorrectly (not as a separate bullet point).

Reply:
Fixed.